reference/language-concepts/augments.markdown

```diff
@@ -30,6 +32,112 @@ Any filenames you put here will appear in the `def.augments_inputs` variable.
 The standard set of masterfiles refers to this variable and will autoload those
 files.
 
+## variables
+
+Variables defined here can target a _namespace_ and or _bundle_ scope explicitly. When defined from `host_specific.json`, variables default to the `main` _bundle_ in the `data` __namespace_ (`$(data:main.MyVariable)`).
```

Comment:
Variables defined in `vars` can also target a namespace and/or a bundle scope explicitly. `variables` only adds the possibility to specify tags and comment. Also the default namespace:bundle for variables is `default:def` and `data:variables` for `def.json` and `host_specific.json`, respectively.

Reply:
Thank you. updated the vars section.